Automattic/vip-go-mu-plugins
# Update/jetpack 9.8

## Description


## Changelog Description

### Plugin Updated: Jetpack 9.8

We upgraded Jetpack 9.8-beta to Jetpack 9.8.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Pin JP by adding `define( 'VIP_JETPACK_PINNED_VERSION', '9.8' );`
In wp-admin navigate  to jetpack and in footer see version 9.8.
